evm/src/contracts/CircleIntegration/Storage.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.22;
+
+/**
+ * struct State {
+ *         // state.slot := 0x0
+ *         uint16 chainId;
```

Comment:
I don't think we should remove getters for these state variables, since this could be a potential breaking change for integrators.

Reply:
The getters can be found in Logic.sol (I added them here since `Logic` inherits `ICircleIntegration). Delegate calls for these should retrieve the now immutable variables in the implementation.

e.g. https://github.com/wormhole-foundation/wormhole-circle-integration/blob/evm/optimize/evm/src/contracts/CircleIntegration/Logic.sol#L163-L166